### PR TITLE
COMMERCE-12514 Implement PayPal Payment Integration and replace old PayPal Payment Method for standard payments

### PR DESCRIPTION
**Pull request by lorenzolapescara**
https://liferay.atlassian.net/browse/COMMERCE-12382
https://liferay.atlassian.net/browse/COMMERCE-12384

Re-sent with automated functional tests from #4476

### CODE REVIEW

**Comment by lorenzolapescara** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONProductsAPI.macro [hunk 1]
```diff
@@ -1366,6 +1366,114 @@ definition {
 		JSONCurlUtil.patch(${curl});
 	}
 
+	@summary = "Use JSON"
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONProductsAPI.macro:1369]: fix summary

**Comment by lorenzolapescara** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
@victoroscuro Failures are not related, the PR can be force-forwarded. Thanks!

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**